mock-up/scss/tech.scss

```diff
@@ -11,29 +12,32 @@
   border-radius: 5px;
   display: flex;
   flex-direction: column;
-  height: 325px;
+  height: 350px;
+  margin: 30px;
+  max-width: 225px;
   padding: 10px;
   position: relative;
-  width: 200px;
+  width: 100%;
 
   &::after {
     content: "";
     box-sizing: border-box;
     border: solid $earls-green 1px;
     border-radius: 5px;
     bottom: 10px;
-    height: 325px;
+    height: 350px;
+    max-width: 225px;
     right: 10px;
     position: absolute;
-    width: 200px;
+    width: 100%;
     z-index: -1;
   }
 }
 
 .tech-img {
   height: 100px;
-  object-fit: cover;
-  width: 120px;
+  object-fit: contain;
```

Comment:
cover

Reply:
The resizing of the image with cover causes that the image won't fit in the width and height and the result is cut it